src/ExpressionEvaluator/Core/Source/FunctionResolver/FunctionResolverBase.cs

```diff
@@ -117,11 +118,10 @@ internal void OnModuleLoad(TProcess process, TModule module, OnFunctionResolvedD
         private bool ShouldHandleRequest(TRequest request)
         {
             var languageId = GetLanguageId(request);
-            if (languageId == Guid.Empty)
-            {
-                return true;
-            }
-            return languageId == LanguageId;
+            return languageId == Guid.Empty ||
+                languageId == LanguageId ||
+                languageId == DkmLanguageId.CausalityBreakpoint ||
+                languageId == DkmLanguageId.MethodId;
```

Comment:
The new BP binder should ignore DkmLanguageId.MethodId.  These are handled by the Causality Monitor.